dev/VSIX/Shared/Resources.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License
+
+using System;
+using System.Globalization;
+using System.Resources;
+
+namespace WindowsAppSDK.TemplateUtilities
+{
+    internal static class Resources
+    {
+        private static readonly Lazy<ResourceManager> _resourceManager =
+            new Lazy<ResourceManager>(() => new ResourceManager("VSPackage", typeof(Resources).Assembly));
+
+        private static ResourceManager ResourceManager => _resourceManager.Value;
```

Comment:
This will be created at the first attempt of reading the string. Can this generate latency for the user? Would be worth it to have it being created before outside of this class and being referenced here?